src/react/components/GenericForm.jsx

```diff
@@ -187,20 +214,26 @@ export class GenericForm extends React.Component {
                     </Space>
                 </Row>
                 );
+            case 'additionalinfo':
```

Comment:
Could 'info' be used instead of adding a new type?

Reply:
Changed this to `info` and removed some padding from Card-component.